pkg/selinux/selinux-client.js

```diff
@@ -128,12 +139,27 @@ export function init(statusChangedCallback) {
                 .then(output => {
                     output = output.split("~~~~~");
                     status.shell = output[0];
+                    let formated_shell = "";
+                    output[0].split("\n").forEach(e => {
+                        const items = e.split(" ");
+                        if (items.length === 2 && items[1] == "-D")
+                            formated_shell += "        semanage " + e + "\n";
+                    });
                     status.modifications = [];
+                    status.ansible = `    - name: Clean all existing selinux rules
+      shell: |
+${formated_shell}`;
```

Comment:
It's not quite obvious to me that we really want this. It's necessary for shell as the `semanage` commands are not idempotent. But the generated Ansible rules already should be. Let's keep this as an open point for discussion for now.

Reply:
This indeed was one of things I wanted to discuss. I added it here so the state after running shell and ansible is the same.